tools/excel/nist/sp-800-53/nist-sp-800-53-rev5_to_iso27001-2022.py

```
@@ -0,0 +1,65 @@
+        temp_df = temp_df.dropna(subset=['target_node_id'])
+
+        # Function to clean values from column A (source_node_id)
+        def clean_source(value):
+            if isinstance(value, str):
+                value = value.strip().lower()                     # Lowercase and remove spaces
+                value = re.sub(r'-(0)(\d)\b', r'-\2', value)      # Remove unnecessary leading zeros (e.g., PX-01 → PX-1)
+                value = re.sub(r'\((0+)(\d+)\)', r'(\2)', value)  # (01) → (1), (003) → (3)
+            return value
+
+        # Function to clean values from column D (target_node_id)
+        def clean_target(value):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)